platform/externalstoragemigration/content_uri_update_script.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+red=$(tput setaf 1)
+green=$(tput setaf 2)
+yellow=$(tput setaf 3)
+default=$(tput sgr0)
+
+STORAGETYPE=$1
+HOST=$2
+USERNAME=$3
+PASSOWORD=$4
+DBNAME=$5
+
+
+
+check_input_value() {
+  if [ -z "$2" ];
+  then
+    echo "$red $(date) $1 is blank or not provided $default"
+    exit 1;
+  elif [ "$1" == "STORAGETYPE" ];
+  then
+    if [[ "$2" != "azure" && "$2" != "s3" ]];
+    then
+    echo "$red $(date) $1 allowed values are azure or s3 $default"
+    exit 1;
+    fi
+  else
+    echo "$green $(date) $1 is provided $default"
+  fi
+}
+
+
+# function to validate database connection
+validate_dbconn() {
+   
+    echo "$green Validating connection to $HOST $default"
+
+    result=$(sqlcmd -S ${HOST} -U ${USERNAME} -P ${PASSOWORD} -d ${DBNAME} -Q "SELECT DB_NAME()" )
+
+    if [ $? -ne 0 ];
+    then
+            echo "$red Error while connecting to SQL Server -> $HOST and Database -> $DBNAME . Please check database details provided"
+            ERROR="${ERROR} \n Database Check Failed: Error while connecting to $1."
+            exit 1;
+    else
+            echo "Connection to $HOST sql server and $DBNAME database established successfully $default"
+    fi
+}
+
+function update_content_uri_azure() {
+
+echo "$green Executing updated content uri in azure in $HOST $default"
+
+STORAGE_HOST=$(kubectl -n uipath get secret aicenter-external-storage-secret -o json | jq '.data.ACCOUNTNAME' | sed -e 's/^"//' -e 's/"$//' | base64 -d)
+
+STORAGE_HOST_SUFFIX=$(kubectl -n uipath get secret aicenter-external-storage-secret -o json | jq '.data.AZURE_FQDN_SUFFIX' | sed -e 's/^"//' -e 's/"$//' | base64 -d)
+
+BUCKET_NAME=$(kubectl -n uipath get secret aicenter-external-storage-secret -o json | jq '.data.BUCKET' | sed -e 's/^"//' -e 's/"$//' | base64 -d)
```

Comment:
Add null/empty check for these three STROAGE_HOST AND BUCKET_NAME

Reply:
done